### PR TITLE
Follow-up edits to PR#2524

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up edits to https://github.com/openshift/openshift-docs/pull/2524

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@tnguyen-rh @bfallonf @tpoitras PTAL :bow:

File: admin_guide/manage_nodes.adoc [hunk 1]
```diff
@@ -271,63 +271,65 @@ on selected nodes.
...
-   means node B is now running both registry pods.
-1. Node B is now marked unschedulable and is evacuated
-1. The service exposing the two pod endpoints on node B, for a brief period of
-   time, looses all endpoints until they are redeployed to node A
+- Node A is marked unschedulable and all pods are evacuated.
+- The registry pod running on that node is now redeployed on node B. This means
+node B is now running both registry pods.
+- Node B is now marked unschedulable and is evacuated.
+- The service exposing the two pod endpoints on node B, for a brief period of
+   time, looses all endpoints until they are redeployed to node A.
```
Comment (bfallonf) [inline admin_guide/manage_nodes.adoc]: s/looses/loses

or maybe

s/looses/loosens

?

File: admin_guide/manage_nodes.adoc [hunk 2]
```diff
@@ -271,63 +271,65 @@ on selected nodes.
...
 
-The same proces using three infrastructure nodes will not result in a service
-disruption.  However, due to pod scheduling, the end result is one node
-running two registry pods, one running one, and another running zero.  The best
-solution is to rely on pod anti-affinity.  This is an alpha feature in
-kubernetes that is available for testing now, but is not yet supported for
-production workloads.
+The same process using three infrastructure nodes does not result in a service
+disruption. However, due to pod scheduling, the end result is one node running
+two registry pods, one running one, and another running zero. The best solution
```
Comment (bfallonf) [inline admin_guide/manage_nodes.adoc]: The "However" sentence doesn't make any sense to me... We say "the end result it one node running two registry pods", but "one running one, and another running zero" only adds up to one. Can this be clarified? cc: @brenton

**Comment by bfallonf** [comment]
@ahardin-rh A couple comments. But good otherwise!

File: admin_guide/manage_nodes.adoc (hunk 2 above)
Comment (brenton) [inline admin_guide/manage_nodes.adoc]: If you have 3 nodes each running one registry and you evacuate/restart one of them the pod will land another another node (which already has a registry).  Therefore when it comes back there will be 2 registry pods running on one node, the node you restarted will have zero registries and the node you haven't touched yet will have zero.  Any way you do it the last node you restart will have zero registries running on it meaning one of the nodes will be running two registries and another will be running one.

**Comment by ahardin-rh** [comment]
@bfallonf @brenton Thanks! I made some edits. PTAL.

File: admin_guide/manage_nodes.adoc [hunk 3]
```diff
@@ -271,63 +271,68 @@ on selected nodes.
...
+- The registry pod running on that node is now redeployed on node B. This means
+node B is now running both registry pods.
+- Node B is now marked unschedulable and is evacuated.
+- The service exposing the two pod endpoints on node B, for a brief period of
+   time, loses all endpoints until they are redeployed to node A.
+
+The same process using three infrastructure nodes does not result in a service
+disruption. However, due to pod scheduling, the end result is one node running
+two registry pods, the node you restarted with zero registries, and the node you
+have not touched yet with zero registries. The last node you restart will have
```
Comment (brenton) [inline admin_guide/manage_nodes.adoc]: The node that hasn't been touched will have 1 registry.

This situation is hard to explain, but it seems a little wordy to me.

**Comment by ahardin-rh** [comment]
@brenton Thanks. Fixed. It is a little verbose, but breaking down the scenario helped my novice brain understand the scenario better. Is there any part that you find non-essential to the explanation? If so, perhaps we can trim it.

File: admin_guide/manage_nodes.adoc [hunk 4]
```diff
@@ -271,63 +271,68 @@ on selected nodes.
...
+- Node B is now marked unschedulable and is evacuated.
+- The service exposing the two pod endpoints on node B, for a brief period of
+   time, loses all endpoints until they are redeployed to node A.
+
+The same process using three infrastructure nodes does not result in a service
+disruption. However, due to pod scheduling, the end result is one node running
+two registry pods, the node you restarted with zero registries, and the node you
+have not touched yet with one registry. The last node you restart will have zero
+registries running on it, so one of the nodes will be running two registries and
+another will be running one. The best solution is to rely on pod anti-affinity.
```
Comment (brenton) [inline admin_guide/manage_nodes.adoc]: This is the sentence that seems mostly redundant to me:

"The last node you restart will have zero registries running on it, so one of the nodes will be running two registries and another will be running one."

We could say something like,

"However, due to pod scheduling, the last Node that is evacuated and brought back in to rotation will be left running zero registries.  The other two nodes will run two and one registries respectively."

**Comment by ahardin-rh** [comment]
@brenton I like it. Edits complete.

**Comment by bfallonf** [comment]
@ahardin-rh Nice work. LGTM.

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/manage_nodes.adoc#admin-guide-manage-nodes[Managing Nodes]
|Added  a new xref:../admin_guide/manage_nodes.adoc#rebooting-nodes[Rebooting Nodes] section.
%

**Pull request merged**